document/FileFormat.mdk

```diff
@@ -1656,8 +1699,9 @@ or include them in bug counts.
 ~
 
 If present, this property conveys the reason or reasons that the result has been suppressed.
-In this version of the SARIF standard, the only supported reason for suppressing a result is that
-the developer has suppressed it in the source code (see &sect;[#result-suppressionStates-suppressedInSource]).
+In this version of the SARIF standard, the only supported reasons for suppressing a result is that
+the developer has suppressed it in the source code (see &sect;[#result-suppressionStates-suppressedInSource])
+or that it is marked as "suppressed" an external database (see &sect;[#result-suppressionStates-suppressedExternally]).
```

Comment:
>suppressed [](start = 25, length = 10)

why do you enquote here and not elsewhere in the para? probably can remove the quotes. might be best to use 'external persisted store' or something more general than database. we could be working with a simple baseline written to a text file, for example. we could use language lile 'external store, such as a database' perhaps.